src/main/java/com/shaft/tools/io/ReportManager.java

```diff
@@ -692,6 +693,23 @@ public static void openAllureReportAfterExecution() {
             new TerminalActions().performTerminalCommand(commandToOpenAllureReport);
         }
     }
+    
+    // TO close running reports so allure results can be cleaned before execution
+ 	// That will work after running test terminated from IDE first.
+ 	public static void closeRunningAllureReport() {
```

Comment:
please remove this code from this PR as it is related to another issue and let's discuss this proposed fix more before we implement it.

Reply:
Ok great.